imports/plugins/included/sitemap-generator/server/no-meteor/schemas/schema.graphql

```diff
@@ -13,7 +13,26 @@ input GenerateSitemapsInput {
 type GenerateSitemapsPayload {
   "The same string you sent with the mutation params, for matching mutation calls with their responses"
   clientMutationId: String
-  
+
   "Whether the sitemap generation job was successfully scheduled"
   wasJobScheduled: Boolean!
 }
+
+extend type Query {
+  "Returns Sitemap object for a shop based on the handle param"
+  sitemap(handle: String!, shopUrl: String!): Sitemap
+}
+
+type Sitemap {
+  "The shop ID"
+  shopId: String!
+
+  "The Sitemap handle, as set in Sitemaps collection"
+  handle: String
```

Comment:
@impactmass `handle` and `xml` should be required

Reply:
good catch 👍 . Updated